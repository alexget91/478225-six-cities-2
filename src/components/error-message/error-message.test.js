import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "./error-message";

it(`Error message correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<ErrorMessage
      error={`error`}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
