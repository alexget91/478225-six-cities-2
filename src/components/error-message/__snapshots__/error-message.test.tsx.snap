// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error message correctly renders after relaunch 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "color": "black",
      "fontSize": "25px",
      "left": 0,
      "margin": "0 auto",
      "padding": "5px",
      "position": "fixed",
      "right": 0,
      "textAlign": "center",
      "top": 0,
      "zIndex": 100,
    }
  }
>
  error
  <span
    className="js-close"
    style={
      Object {
        "cursor": "pointer",
        "float": "right",
      }
    }
  >
    [close]
  </span>
</div>
`;
